api/lib/db.js

```diff
@@ -201,6 +201,170 @@ var DB = function(config) {
 
   };
 
+
+  self.getAccountTxSeq = function (options, callback) {
+    var result = prepareQuery();
+    if (result.error) {
+      callback(result);
+      return;
+    }
+    
+    if (options.sequence)
+      log.debug(new Date().toISOString(), 'getting transaction:', options.account, options.sequence);
+    else
+      log.debug(new Date().toISOString(), 'getting transactions:', options.account);
+
+    //execute the query      
+    result.query.nodeify(function(err, rows) {
+
+      if (options.sequence)
+        log.debug(new Date().toISOString(), 'transaction found'); 
+      else
+        log.debug(new Date().toISOString(), (rows ? rows.length : 0) + ' transactions found'); 
+
+            
+      if (err) {
+        log.error(err);
+        callback({error:err, code:500});
+        return;
+        
+      //get a count of all the rows that
+      //are found without a limit
+      } else if (rows.length) {
```

Comment:
only do this if options.sequence was not specified - we already know its 1 in that case